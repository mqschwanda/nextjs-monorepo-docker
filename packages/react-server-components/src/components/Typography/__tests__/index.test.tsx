import { render } from '@testing-library/react';
import { Typography } from '@';

describe('@mqs/react-server-components', () => {
  describe('components', () => {
    describe('<Typography />', () => {
      it('renders', () => {
        const testId = Typography.name;
        const children = 'children...';

        const { getByTestId, asFragment } = render((
          <Typography
            className='test'
            testId={testId}
          >
            { children }
          </Typography>
        ));

        const element = getByTestId(testId);
        expect(element).toBeTruthy();

        expect(asFragment()).toMatchSnapshot();
      });
    });
  });
});
