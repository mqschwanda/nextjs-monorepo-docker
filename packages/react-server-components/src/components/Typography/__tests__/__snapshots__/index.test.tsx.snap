// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/react-server-components components <Typography /> renders 1`] = `
<DocumentFragment>
  <span
    class="test Typography Typography_variant_body1"
    data-testid="Typography"
  >
    children...
  </span>
</DocumentFragment>
`;
